packages/grafana-ui/src/utils/mousetrap.ts

```diff
@@ -0,0 +1,18 @@
+import Mousetrap from 'mousetrap';
+
+// @ts-ignore
+import enableMouseTrapPause from 'mousetrap-pause';
+import 'mousetrap-global-bind';
+
+let mousetrapInstance: MousetrapStatic & {
+  pause: () => void;
+  unpause: () => void;
+};
+
+export const getMousetrap = () => {
```

Comment:
Not sure about this name at all. think `keybindings` would work better. But then it's a function, `getKeybindings`?